deeprankcore/dataset.py

```diff
@@ -297,30 +207,31 @@ def load_one_graph(self, fname, mol): # noqa
                 edge_index = torch.empty((2, 0), dtype=torch.long)
             edge_index = edge_index.to(self.device)
 
-            # edge feature (same issue as above)
-            if self.edge_feature is not None and len(self.edge_feature) > 0 and \
-               Efeat.EDGE in grp:
-
+            # edge feature
+            # we have to have all the edges i.e : (i,j) and (j,i)
+            if (self.edge_features is not None 
+                    and len(self.edge_features) > 0 
+                    and Efeat.EDGE in grp):
                 edge_data = ()
-                for feat in self.edge_feature:
+                for feat in self.edge_features:
                     if feat[0] != '_':   # ignore metafeatures
                         vals = grp[f"{Efeat.EDGE}/{feat}"][()]
                         if vals.ndim == 1:
                             vals = vals.reshape(-1, 1)
                         edge_data += (vals,)
                 edge_data = np.hstack(edge_data)
                 edge_data = np.vstack((edge_data, edge_data))
-                edge_data = self.edge_feature_transform(edge_data)
+                edge_data = self.edge_features_transform(edge_data)
                 edge_attr = torch.tensor(edge_data, dtype=torch.float).contiguous()
             else:
                 edge_attr = torch.empty((edge_index.shape[1], 0), dtype=torch.float).contiguous()
             edge_attr = edge_attr.to(self.device)
 
             if any(key in grp for key in ("internal_edge_index", "internal_edge_data")):
                 warnings.warn(
-                    """Internal edges are not supported anymore.
-                    You should probably prepare the hdf5 file
-                    with a more up to date version of this software.""", DeprecationWarning)
+                    """Internal edges are not supported anymore.\n
+                    You should probably prepare hdf5 files with a more up to date version of this software.""", 
+                    DeprecationWarning)
```

Comment:
do we still need this warning or can it go?

Reply:
We can leave it for now, just in case. This warning was placed by Coos for older hdf5 files.

Reply:
@cbaakman is this warning still necessary?